testsuite/forge-cli/src/main.rs

```diff
@@ -452,9 +452,13 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
             })),
         "config" => config.with_network_tests(vec![&ReconfigurationTest]),
         "network_partition" => config.with_network_tests(vec![&NetworkPartitionTest]),
-        "network_latency" => config
-            .with_network_tests(vec![&NetworkLatencyTest])
-            .with_success_criteria(SuccessCriteria::new(4000, 10000, true, None, None)),
+        "three_region_simulation" => config
+            .with_initial_validator_count(NonZeroUsize::new(20).unwrap())
+            .with_initial_fullnode_count(10)
+            .with_emit_job(EmitJobRequest::default().mode(EmitJobMode::ConstTps { tps: 5000 }))
```

Comment:
Curious, did you try the mempool backlog load and what was the result from that?

Reply:
Mempool backlog seemed to result in less reproducible results, which makes it hard for us to set a good SuccessCritiera TPS/latency too. I think the key is we just land this and start running it continuously, and tune from there.